src/actions-types/PasswordActionsType.js

```diff
@@ -0,0 +1,18 @@
+export const FORGOT_PASSWORD_INITIALIZED = 'FORGOT_PASSWORD_INITIALIZED';
```

Comment:
@luc-tuyishime, why do you have this file named in pascal case: `src/actions-types/PasswordActionsType.js`, it should be in camel case: `src/actions-types/passwordActionsType.js`

Reply:
this file no longer exist because now we have a file for user action-types where we initialize all the action-types.